server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -757,7 +758,7 @@ private void configureQualifiedHostName(String qualifiedHostName, String provide
                 // Give up
                 qualifiedHostName = "unknown-host.unknown-domain";
             } else {
-                qualifiedHostName = qualifiedHostName.trim().toLowerCase();
+                qualifiedHostName = qualifiedHostName.trim().toLowerCase(Locale.ENGLISH);
```

Comment:
This should be removed or, alternatively, use Locale.getDefault(). It is the same case as the Host Controller variant.

Reply:
👍 Fixed.